core/src/main/java/org/arquillian/smart/testing/SystemPropertyResolver.java

```diff
@@ -0,0 +1,33 @@
+package org.arquillian.smart.testing;
+
+public class SystemPropertyResolver {
```

Comment:
If we are using `System.getProperty("foo.bar")` at multiple places. So instead of using it, isn't it a good idea to have it at one place & use it from there to follow DRY, and for less maintenance.

If we are using at only one place then we can use `System.getProperty`.